library/java/io/envoyproxy/envoymobile/engine/BUILD

```diff
@@ -0,0 +1,18 @@
+licenses(["notice"])  # Apache 2
+
+# These source files must be re-exported to the kotlin custom library rule to ensure their
+# inclusion.
+filegroup(
```

Comment:
Ah! This is how to achieve this

Reply:
Yep, it also means if we ever add more java srcs, we don't have to worry about missing them in the kotlin rule. :)